python-checks/src/main/java/org/sonar/python/checks/cdk/UnrestrictedAdministrationCheckPartCfnSecurity.java

```diff
@@ -0,0 +1,185 @@
+/*
```

Comment:
I'm not sure what I should review in this class: was it just copy-pasted from `UnrestrictedAdministrationCheck`?

Reply:
I confirm, no review required as it is indeed a copy/paste, I put this in the original commit message :

> Not necessary to review file [UnrestrictedAdministrationCheckPartCfnSecurity.java](https://github.com/SonarSource/sonar-python/pull/1239/files#diff-7bffa4cdc6308635fe43979de897f7756575feff19fd5ead9d2621958bbd3586), the content has been moved so it appears as new, but in fact It has been already reviewed in the https://github.com/SonarSource/sonar-python/pull/1229.